app/src/main/java/fr/free/nrw/commons/media/MediaDetailFragment.java

```diff
@@ -262,29 +306,31 @@ private void displayMediaDetails() {
      * The imageSpacer is Basically a transparent overlay for the SimpleDraweeView
      * which holds the image to be displayed( moreover this image is out of
      * the scroll view )
-     * @param imageInfo used to calculate height of the ImageSpacer
+     * @param scrollWidth the current width of the scrollView
      */
-    private void updateAspectRatio(ImageInfo imageInfo) {
-        if (imageInfo != null) {
-            int screenWidth = scrollView.getWidth();
-            int finalHeight = (screenWidth*imageInfo.getHeight()) / imageInfo.getWidth();
+    private void updateAspectRatio(int scrollWidth) {
+        if (imageData != null) {
+            int finalHeight = (scrollWidth*imageData.getHeight()) / imageData.getWidth();
             ViewGroup.LayoutParams params = image.getLayoutParams();
             ViewGroup.LayoutParams spacerParams = imageSpacer.getLayoutParams();
             params.height = finalHeight;
             spacerParams.height = finalHeight;
             image.setLayoutParams(params);
             imageSpacer.setLayoutParams(spacerParams);
+            imageLandscape.setLayoutParams(params);
         }
     }
 
     private final ControllerListener aspectRatioListener = new BaseControllerListener<ImageInfo>() {
         @Override
         public void onIntermediateImageSet(String id, @Nullable ImageInfo imageInfo) {
-            updateAspectRatio(imageInfo);
+            imageData = imageInfo;
```

Comment:
I would name imageData as imageInfo, considering its class type and the other usages in the code.